addons/l10n_jp/data/l10n_jp_chart_data.xml

```diff
@@ -10,4 +12,93 @@
         <field name="currency_id" ref="base.JPY"/>
     </record>
 
+    <!-- Account Tags -->
```

Comment:
@AdrienHorgnies We probably just need to ignore the changes in this file.

Reply:
indeed. I removed them.